index.html

```diff
@@ -2,21 +2,130 @@
+        <article>
+          <h2 id="article-id-03" class="article-title article-title-three">Folklore</h2>
+          <div class="article-figures-container">
+            <figure>
+              <img class="article-image" src="images/image05.jpg" alt="image alt text">
+              <figcaption class="article-image-caption">image caption</figcaption>
+              <cite><a href="#">citation</a></cite>
+            </figure>
+            <figure>
+              <img class="article-image" src="images/image06.jpg" alt="image alt text">
+              <figcaption class="article-image-caption">image caption</figcaption>
+              <cite><a href="#">citation</a></cite>
+            </figure>
+          </div>
+          <p class="article-summary">
+            Thai folklore holds the Phaya nagas to be semi-divine, demi-creatures, which possess supernatural powers as has been described in Buddhist and Hindu cosmology. 
+            The "Kamchanod Forest" Ban Dung district, Udon Thani province, which is held in high reverence and fear across Thailand, is believed to be the border between the human world and the netherworld, and is frequently depicted in Thai folklore as the site of many hauntings, but more frequently is considered to be the home of the Nāga.
+          </p>
+        </article>
+        <article id="article-id-04">
+          <h2 class="article-title">A Naga tries to Ordain</h2>
+          <div class="article-figures-container">
+            <figure>
+              <img class="article-image" src="images/image07.jpg" alt="image alt text">
+              <figcaption class="article-image-caption">image caption</figcaption>
+              <cite><a href="#">citation</a></cite>
+            </figure>
+            <figure>
+              <img class="article-image" src="images/image08.png" alt="image alt text">
+              <figcaption class="article-image-caption">image caption</figcaption>
+              <cite><a href="#">citation</a></cite>
+            </figure>
+          </div>
+          <p class="article-summary">The Buddhist nāga generally has sometimes portrayed as a human being with a snake or dragon extending over his head. One nāga, in human form, attempted to become a monk; and when telling it that such ordination was impossible, the Buddha told it how to ensure that it would be reborn a human, and so able to become a monk.
+            The nagas are believed to both live on Nagaloka, among the other minor deities, and in various parts of the human-inhabited earth. Some of them are water-dwellers, living in streams or the ocean; others are earth-dwellers, living in caverns.
+            The nagas are the followers of Virūpākṣa (Pāli: Virūpakkha), one of the Four Heavenly Kings who guards the western direction. They act as a guard upon Mount Sumeru, protecting the dēvas of Trāyastriṃśa from attack by the asuras.</p>
+        </article>
+      </main>
+      <footer>
+        <h3 class="footer-title">Footer</h3>
```

Comment:
The h3 here means that you are nesting "Footer" under "A Naga tries to Ordain" on line 103